pandas/tests/groupby/test_categorical.py

```diff
@@ -508,7 +508,7 @@ def test_datetime():
     desc_result = grouped.describe()
 
     idx = cats.codes.argsort()
-    ord_labels = cats.take_nd(idx)
+    ord_labels = cats.take(idx)  # TODO: deprecate take<->take_nd alias
```

Comment:
can you make an issue (and remove the comment)

Reply:
done